server/opts.go

```diff
@@ -331,6 +331,8 @@ type Options struct {
 	JetStreamLimits       JSLimitOpts
 	JetStreamTpm          JSTpmOpts
 	JetStreamMaxCatchup   int64
+	StreamMaxBufferedMsgs int
```

Comment:
Do we need explicit json annotations here? What is the default name? Does it move camel case breaks to `_`?

Reply:
The default JSON name would be the variable name exactly, but as I mentioned above there's a comment on that srtuct saying the JSON tags are deprecated as that struct is no longer used for monitoring, so I think it's irrelevant?

Reply:
We should be consistent, either mark it as non exported or pick a name in case it is IMO. I prefer camel case -> snake case.